src/providers/wms/qgswmscapabilities.cpp

```diff
@@ -1077,6 +1075,39 @@ void QgsWmsCapabilities::parseDimension( const QDomElement &element, QgsWmsDimen
   dimensionProperty.extent = element.text();
 }
 
+void QgsWmsCapabilities::parseExtent( const QDomElement &element, QVector<QgsWmsDimensionProperty> &dimensionProperties )
+{
+  const QString name = element.attribute( QStringLiteral( "name" ) );
+  // try to find correponsding dimension property -- i.e. we upgrade the WMS 1.1 split of Dimension and Extent to 1.3 style where Dimension holds the extent information
+  for ( auto it = dimensionProperties.begin(); it != dimensionProperties.end(); ++it )
+  {
+    if ( it->name == name )
+    {
+      it->extent = element.text();
+
+      it->defaultValue = element.attribute( QStringLiteral( "default" ) );
+
+      if ( !element.attribute( QStringLiteral( "multipleValues" ) ).isNull() )
+      {
+        QString multipleValuesAttribute = element.attribute( QStringLiteral( "multipleValues" ) );
+        it->multipleValues = ( multipleValuesAttribute == QLatin1String( "1" ) || multipleValuesAttribute == QLatin1String( "true" ) );
```

Comment:
```suggestion
        it->multipleValues = ( multipleValuesAttribute == '1' ) || multipleValuesAttribute == QLatin1String( "true" ) );
```

Reply:
Actually there's no QString operator==( QChar ) overload, so this would use the `QString( QChar() )` implicit constructor and end up calling `operator==( QString )`. Eeek - we're actually better off leaving it at QLatin1String to utilise the optimised equality operator.